lib/node_modules/@stdlib/math/base/special/fibonacci/README.md

```diff
@@ -137,6 +137,95 @@ for ( i = 0; i < 79; i++ ) {
 
 <!-- /.examples -->
 
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/math/base/special/fibonacci.h"
+```
+
+#### stdlib_base_fibonacci( n )
+
+Computes the nth [Fibonacci number][fibonacci-number].
+
+```c
+int64_t out = stdlib_base_fibonacci( 0 );
+// returns 0
+
+out = stdlib_base_fibonacci( 1 );
+// returns 1
+```
+
+The function accepts the following arguments:
+
+-   **n**: `[in] int32_t` input value.
+
+```c
+int64_t stdlib_base_fibonacci( const int32_t n );
```

Comment:
Here as we can have `NaN` as input value, so we will need `n` to be of `double` type right?

Reply:
Actually, I think limiting `n` to `int32_t` should be fine. The implementation would omit the `NaN` check and we'd remove any associated tests in `test.native.js`.

Reply:
We have precedent elsewhere of not accepting/returning doubles, thus deviating from JS. Here, I think the main thing is that the range of output values should match the JS implementation. The input dtype can be restricted to `int32_t`.

Reply:
Okay, will make necessary changes.